internal/api/handlers.go

```diff
@@ -489,6 +510,7 @@ func logErrorAndWriteResponse(msg string, status int, w http.ResponseWriter) {
 	log.Println(errMsg)
 	w.WriteHeader(status)
 	if _, err := w.Write([]byte(errMsg)); err != nil {
+		// TODO: how did this get merged?
```

Comment:
Should we fix this right away? It seems like a potential infinite loop that could crash the server.

Otherwise, is there a ticket to track this?

Reply:
fixed. lets not leave this here haha.